### ISSUE TITLE
Product specification attributes, If we select attribute type as Custom HTML Text, Value field should be HTML Editor

### ISSUE THREAD

**Issue by balureddy003** [grandnode/grandnode2#200]
- Product specification attributes, If we select attribute type as Custom HTML Text, the Value field should be HTML Editor
- This change gives more flexibility to customize specifications in the product view.
- Let me know if I can contribute to this change

![image](https://user-images.githubusercontent.com/7783528/145705021-72d9e33a-39b7-4b5b-a495-436555cf03a0.png)

**Comment by KrzysztofPajak** [comment]
@balureddy003 
Yes, it should be converted. If you can do this you can contribute it.
